service/task.go

```diff
@@ -891,9 +891,12 @@ func (uis *UIServer) testLog(w http.ResponseWriter, r *http.Request) {
 	uis.render.Stream(w, http.StatusOK, data, "base", "task_log.html")
 }
 
-func (uis *UIServer) getTestResults(w http.ResponseWriter, r *http.Request, projCtx projectContext, uiTask *uiTaskData) []task.TestResult {
+func (uis *UIServer) getTestResults(projCtx projectContext, uiTask *uiTaskData) []task.TestResult {
 	if err := projCtx.Task.PopulateTestResults(); err != nil {
-		uis.LoggedError(w, r, http.StatusInternalServerError, err)
+		grip.Error(message.WrapError(err, message.Fields{
+			"task_id": projCtx.Task.Id,
+			"message": "fetching test results for task",
+		}))
 		return nil
```

Comment:
Should we initialize uis.TestResults above? (Is there concern around nil pointer errors if we aren't sending an error? Didn't read back where this is used though.

Reply:
The only place it gets used in Go code is as part of append inside that function. After that I _think_ it goes to JS code, which could have an issue with null maybe? 

I don't see any JS errors in the console but all I'm doing is loading the task page and that seems to work.

Reply:
Happy to change it though if it seems like a good idea.